openassessment/xblock/test/test_submission.py

```diff
@@ -445,6 +451,56 @@ def test_open_saved_response(self, xblock):
             }
         )
 
+        self.assertEqual(2, len(xblock.get_file_descriptions()))
```

Comment:
Nit: we should assert that the context of the list equals what we expect.  We should also do the same for the file names and the file sizes.